[2023-05-05 05:36:51,707] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_data_from_api_dag.fetch_and_save manual__2023-05-05T05:36:45.506868+00:00 [queued]>
[2023-05-05 05:36:51,716] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_data_from_api_dag.fetch_and_save manual__2023-05-05T05:36:45.506868+00:00 [queued]>
[2023-05-05 05:36:51,717] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-05-05 05:36:51,717] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2023-05-05 05:36:51,718] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-05-05 05:36:51,729] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): fetch_and_save> on 2023-05-05 05:36:45.506868+00:00
[2023-05-05 05:36:51,732] {standard_task_runner.py:52} INFO - Started process 6644 to run task
[2023-05-05 05:36:51,735] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ingest_data_from_api_dag', 'fetch_and_save', 'manual__2023-05-05T05:36:45.506868+00:00', '--job-id', '177', '--raw', '--subdir', 'DAGS_FOLDER/sample_from_api.py', '--cfg-path', '/tmp/tmpbwgbg_3p', '--error-file', '/tmp/tmpwihablmq']
[2023-05-05 05:36:51,737] {standard_task_runner.py:77} INFO - Job 177: Subtask fetch_and_save
[2023-05-05 05:36:51,814] {logging_mixin.py:109} INFO - Running <TaskInstance: ingest_data_from_api_dag.fetch_and_save manual__2023-05-05T05:36:45.506868+00:00 [running]> on host ccbba2675476
[2023-05-05 05:36:51,855] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/xcom.py:323 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2023-05-05 05:36:51,870] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/renderedtifields.py:126 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2023-05-05 05:36:51,880] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/renderedtifields.py:162 SAWarning: Coercing Subquery object into a select() for use in IN(); please pass a select() construct explicitly
[2023-05-05 05:36:51,891] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/renderedtifields.py:163 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2023-05-05 05:36:51,896] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ingest_data_from_api_dag
AIRFLOW_CTX_TASK_ID=fetch_and_save
AIRFLOW_CTX_EXECUTION_DATE=2023-05-05T05:36:45.506868+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-05T05:36:45.506868+00:00
[2023-05-05 05:36:52,034] {python.py:175} INFO - Done. Returned value was: None
[2023-05-05 05:36:52,054] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ingest_data_from_api_dag, task_id=fetch_and_save, execution_date=20230505T053645, start_date=20230505T053651, end_date=20230505T053652
[2023-05-05 05:36:52,117] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-05-05 05:36:52,163] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-05-05 05:36:52,164] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/dag.py:1076 SADeprecationWarning: Query.value() is deprecated and will be removed in a future release.  Please use Query.with_entities() in combination with Query.scalar() (deprecated since: 1.4)
[2023-05-05 05:36:52,165] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/dag.py:1076 SADeprecationWarning: Query.values() is deprecated and will be removed in a future release.  Please use Query.with_entities() (deprecated since: 1.4)
